src/modules/dashboard/components/ExpenditurePage/Stages/LinkedMotions/LinkedMotions.tsx

```diff
@@ -52,24 +78,22 @@ const LinkedMotions = ({ status, motionLink, motion, id }: Props) => {
         </div>
       </div>
       <div className={styles.statusWrapper}>
-        {formatMessage(MSG.foundExp, { motion, id })}
-        {status === MotionStatus.Pending && motionLink ? (
+        {motionLink ? (
           <Link to={motionLink} className={styles.link}>
-            <FormattedMessage {...MSG.motion} />
+            {formatMessage(MSG.foundExp, { motion, id })}
```

Comment:
Why did you switch to formatMessage here? It's preferable to keep `FormattedMessage` where possible.